<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlotGeneratorClass</class>
 <widget class="QMainWindow" name="PlotGeneratorClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1118</width>
    <height>643</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PlotGenerator</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="RenderArea" name="renderArea" native="true"/>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Tab 1</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="pushButton_load_bcf">
          <property name="text">
           <string>Load BCF file</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_save_plot">
          <property name="text">
           <string>Save plot file</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Screen</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>W</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_screen_width">
             <property name="text">
              <string>100</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>H</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_screen_height">
             <property name="text">
              <string>100</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>Plot Info</string>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string># Entities</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="label_num_entities">
             <property name="text">
              <string>0</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Width</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="label_plot_width">
             <property name="text">
              <string>0</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Height</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLabel" name="label_plot_height">
             <property name="text">
              <string>0</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string>Plot Control</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="0" column="0">
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>dX</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="lineEdit_dx">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="maximumSize">
                <size>
                 <width>100</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>0</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>dY</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLineEdit" name="lineEdit_dy">
               <property name="maximumSize">
                <size>
                 <width>100</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>0</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_8">
               <property name="text">
                <string>Theta</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QLineEdit" name="lineEdit_theta">
               <property name="maximumSize">
                <size>
                 <width>100</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>0</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Tab 2</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1118</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>RenderArea</class>
   <extends>QWidget</extends>
   <header>RenderArea.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="PlotGenerator.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton_load_bcf</sender>
   <signal>clicked()</signal>
   <receiver>PlotGeneratorClass</receiver>
   <slot>load_bcf_file()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1053</x>
     <y>84</y>
    </hint>
    <hint type="destinationlabel">
     <x>906</x>
     <y>36</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_screen_width</sender>
   <signal>returnPressed()</signal>
   <receiver>PlotGeneratorClass</receiver>
   <slot>update_screen()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>989</x>
     <y>163</y>
    </hint>
    <hint type="destinationlabel">
     <x>1084</x>
     <y>35</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_screen_height</sender>
   <signal>returnPressed()</signal>
   <receiver>PlotGeneratorClass</receiver>
   <slot>update_screen()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1065</x>
     <y>158</y>
    </hint>
    <hint type="destinationlabel">
     <x>1112</x>
     <y>44</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_dx</sender>
   <signal>returnPressed()</signal>
   <receiver>PlotGeneratorClass</receiver>
   <slot>plot_edited()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1071</x>
     <y>307</y>
    </hint>
    <hint type="destinationlabel">
     <x>1117</x>
     <y>307</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_dy</sender>
   <signal>returnPressed()</signal>
   <receiver>PlotGeneratorClass</receiver>
   <slot>plot_edited()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1074</x>
     <y>332</y>
    </hint>
    <hint type="destinationlabel">
     <x>1114</x>
     <y>337</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_save_plot</sender>
   <signal>clicked()</signal>
   <receiver>PlotGeneratorClass</receiver>
   <slot>save_plot_file()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1078</x>
     <y>116</y>
    </hint>
    <hint type="destinationlabel">
     <x>1114</x>
     <y>126</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_theta</sender>
   <signal>returnPressed()</signal>
   <receiver>PlotGeneratorClass</receiver>
   <slot>plot_edited()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1071</x>
     <y>355</y>
    </hint>
    <hint type="destinationlabel">
     <x>1114</x>
     <y>368</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>load_bcf_file()</slot>
  <slot>update_screen()</slot>
  <slot>plot_edited()</slot>
  <slot>save_plot_file()</slot>
 </slots>
</ui>
